src/objectTranslations/Contact-ca.objectTranslation

```diff
@@ -3471,66 +3471,38 @@
         <help>Quin número de telèfon s&apos;hauria d&apos;utilitzar per a la majoria de comunicacions que incloguin aquest contacte?</help>
         <label>Telèfon preferit</label>
         <name>PreferredPhone__c</name>
-        <picklistValues>
```

Comment:
Ryan i think we are going to have problems here too.

Reply:
I actually wanna pull @skosgi1 into this bit. Can you verify if this is what it is supposed to be? I feel like this is going to break translation work that we did.

Reply:
@lifewithryan Sorry noticing this just now...looking into it...FYI @robertbwatson please don't merge this yet.